app/styles/mixins/_layout.scss

```diff
@@ -0,0 +1,6 @@
+@mixin l-bottom {
```

Comment:
So in this example, there could be an alert__header and an alert__header--success or an alert and alert--success?